tests/integration/aliases/nodejs/adopt_into_component/step1/package.json

```diff
@@ -1,8 +1,6 @@
 {
     "name": "aliases",
     "license": "Apache-2.0",
-    "main": "bin/index.js",
-    "typings": "bin/index.d.ts",
```

Comment:
What will be the impact for users if they have a `main` leftover in their `package.json` that isn't currently used? Do we need to call out anything else explicitly in the changelog so users are aware?

Reply:
@justinvp I've already called this out in the changelog, but since you and others have asked, it's worth mentioning this:

1. Largely, users with a `package.json` outside the Pulumi root directory could not run their programs before this change. So nobody who currently has a working program would be affected by this change.
2. This change increases the set of programs that Pulumi accepts to include those with a `package.json` outside of the Pulumi root directory. For these programs, we use the behavior [as defined by NPM](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#main). (Yarn [does not offer guidance](https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-main).) In short, **the `main` file is relative to the directory containing `package.json`.** Previously, this was the same as saying the Pulumi `root` directory.